snntorch/_neurons/leaky.py

```diff
@@ -141,6 +141,7 @@ def __init__(
         output=False,
         graded_spikes_factor=1.0,
         learn_graded_spikes_factor=False,
+        reset_delay=True,
```

Comment:
as a clarifying comment: @jeshraghian and I discussed this a while ago - this flag is needed to match up the neuron dynamics with NIR

Reply:
The latest version of the Leaky neuron had some minor refactoring to get it compatible with torch.compile(), so there will be some conflicts. I'll iron these out today.